rholang/reference_doc/sends_and_receives/README.md

```diff
@@ -1,17 +1,62 @@
 ## Sends and Receives
 
-### Tuple space
-- Concrete, technical explanation of the tuple space
+### Tuplespace (RSpace)
+Rholang is based on the rhoVM whose computational model is centered around processes sending messages to one another to synchronize when necessary. These operations are coordinated through the tuplespace.
+Although it is unique from the Turing or Von Neuman architectures that many programmers will be familiar with, it is quite understandable. All tuplespace operations can be built up from three building blocks.
+
+![building blocks](buildingBlocks.png)
+
+* A channel on which messages are send and received
+* Data to be sent (includes [primitive types](../primatives/README.md), [data structures](../data_structures/README.md), or even entire processes. )
+* Continuations which spawn new processes when a message is received
 
 ### Sends
-- Content that can be sent
-- Difference between one-tuple and single process in the syntax
-- Persistent sends
-- Send a process (receive a name)
+![Sends](send.png)
+Sending a name associates the sent data with the channel in the tuplespace. It will wait there forever until it matches with a receive on the same channel.
+
+#### Single Send
+`chan!(data)`
+Sends data which must be a process on the channel chan.
+
+#### Persistent Send
+`chan!!(data)`
+This send will never be consumed. While arbitrarily many recieves can receive the data, it will still remain in the tuplespace.
+
+#### Polyadic Sends
+The simple sends we've described so far are sufficient to make rholang Turing complete, but it is often convenient to send multiple...
 
 ### Receives
-- Patterns in a `for`
-- Persistent send
-- Contracts as sugar for a persistent send w/specific type of pattern
-- Single send
-- (send a process) Receive a name 
+![Receive](receive.png)
+
+A receive associates a continuation with a channel in the tuplespace.
```

Comment:
This is an insightful way of thinking about receives in the tuple space from a K Framework/type system/formal verification point of view. (@Isaac-DeFrain)